zUtilities/QuickSave.cpp

```diff
@@ -59,10 +59,88 @@ namespace GOTHIC_ENGINE {
       return;
     }
 
-    iLastSaveSlot++;
-    iLastSaveNumber++;
-    if ( iLastSaveSlot > Options::MaxSaveSlot || iLastSaveSlot < Options::MinSaveSlot )
-      iLastSaveSlot = Options::MinSaveSlot;
+    if ( Options::UseQuickSave ) {
```

Comment:
Should check for `Options::UseBinarySave`

Reply:
changed `Options::UseBinarySave` -> `Options::UseQuickSave`